manifests/0000_26_capi-operator_11_deployment.yaml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-capi-operator
+  namespace: openshift-cluster-api
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/delete: "true"
+  labels:
+    k8s-app: cluster-capi-operator
+spec:
+  selector:
+    matchLabels:
+      k8s-app: cluster-capi-operator
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        k8s-app: cluster-capi-operator
+    spec:
+      priorityClassName: system-node-critical
```

Comment:
Why is this component system-node-critical?

Reply:
I'll remove, just from copy/paste